### PR TITLE
Improve cli

### PR DESCRIPTION
**Pull request by FynnBe**
set __version__ attribute to package version (was model format version before) and print this package version when using the CLI

### CODE REVIEW

File: bioimageio/spec/__init__.py [hunk 1]
```diff
@@ -16,6 +20,6 @@
 save_raw_node = utils.IO.save_raw_node
 serialize_raw_node_to_dict = utils.IO.serialize_raw_node_to_dict
 
-__version__ = _get_args(ModelFormatVersion)[-1]
-
 # autogen: stop
+
+__version__ = yaml.load(pathlib.Path(__file__).parent / "VERSION")["version"]
```
Comment (oeway) [inline bioimageio/spec/__init__.py:25]: It's a JSON file, so we should use `json.load`, or would `yaml.load` also work?

Comment (FynnBe) [reply bioimageio/spec/__init__.py:25]: as it happens to also be valid yaml syntax `yaml.load` also works... But if it fundamentally is a json I'll better change it already...

Comment (oeway) [reply bioimageio/spec/__init__.py:25]: Interesting, so yaml is a super set of json? didn't know you can load it with yaml.

Comment (FynnBe) [reply bioimageio/spec/__init__.py]: almost. there are some differences.. https://stackoverflow.com/a/21585224

**Review by oeway — APPROVED** [review]
Looks good then, thanks!

**Pull request merged**